chromium/popup.html

```diff
@@ -60,7 +60,8 @@ <h3 i18n="chrome_experimental_rules"></h3>
 </div>
 
 <footer>
-    <a id="aboutTitle" href="https://www.eff.org/https-everywhere" target="_blank" i18n="about_title">blah</a>
+    <a id="aboutTitle" href="https://www.eff.org/https-everywhere" target="_blank" i18n="about_title">blah</a><br><br>
```

Comment:
Either leave this blank or fill in the default English value, rather than using "blah"